packages/node_modules/@webex/webex-core/src/interceptors/payload-transformer.js

```diff
@@ -36,6 +36,10 @@ export default class PayloadTransformerInterceptor extends Interceptor {
    * @returns {Object}
    */
   onResponse(options, response) {
+    if (options.disableTransform) {
+      return response;
+    }
```

Comment:
To confirm, in the case that this option is passed, we simply skip encryption flow that follows this conditional?

Reply:
correct